app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Users::SessionsController < Devise::SessionsController
+  skip_before_action :verify_authenticity_token
+
+  # POST /resource/sign_in
+  def create
+    resource = warden.authenticate!(:scope => resource_name, :recall => "#{controller_path}#new")
+    set_flash_message(:notice, :signed_in) if is_navigational_format?
+    sign_in(resource_name, resource)
+    respond_to do |format|
+      format.html { respond_with resource, :location => after_sign_in_path_for(resource) }
+      format.json {
+        return render json: {
+          success: true,
```

Comment:
ist `success` hier wirklich immer `true` und gibt es immer eine richtige `resource`?

Reply:
resource is user. wenn email und pw falsch bricht es glaub ich bei der sign_in methode vorher schon ab. Die Frage ist ob das dann auch als json Antwort passiert. Wahrscheinlich nicht.

Reply:
Error-Meldungen kommen im Detail nicht im JSON an, lediglich ein Error-Status. Das ist aber auch ok, da im Detail nicht interessant ist ob es die E-Mail nicht gab oder das Passwort falsch war. Außerdem ist es auch sichheitstechnisch besser diese Detailinformation nicht zu geben.